app/src/main/res/animator/raise.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item
+            android:state_enabled="true"
+            android:state_pressed="true"
+            >
+        <objectAnimator
+                android:duration="@android:integer/config_shortAnimTime"
+                android:propertyName="translationZ"
```

Comment:
It does not work.
android:stateListAnimator is ignored on pre-lollipop devices.
Probably you know. I think that it is difficult to equalize .
But pre lollipop selector is working.
![4 4](https://cloud.githubusercontent.com/assets/1386930/22657755/eab39d3c-ecda-11e6-89bc-d1b6dc02d845.gif)